contents/handbook/growth/sales/refunds.md

```diff
@@ -40,13 +40,57 @@ Subsequent incidents:
 Refund requests can come through different channels:
 
 In-app ticket
-- Comms team assigns these requests to RevOps (Mine).
-- RevOps reviews the request, issues refund or credits based on the eligibility and criteria outlined above, and responds to customer.
+- Comms team reviews the request, issues refund or credits based on the eligibility and criteria outlined above, and responds to customer.
 
 Contact sales form or email to sales@posthog.com
-Account Executives can direct these to RevOps using one of the two options below:
-- Slack option: AEs use the ticket emoji in the #website-contact-sales Slack channel to auto-create a Zendesk ticket, which is then assigned to RevOps for processing.
-- Salesforce option: AEs move the lead created from this request to Mine, who reviews and processes it according to criteria.
+Account Executives can direct these to comms team using the ticket emoji in the #website-contact-sales Slack channel to auto-create a Zendesk ticket
 
 Large account requests
-- For large accounts managed by an AE, the AE leads the customer conversation and requests RevOps to issue the refund in Stripe.
+- For large accounts managed by an AE, AE may lead the customer conversation and can loop in Comms or RevOps team as needed to process credits or refund in Stripe.
+
+
+## Refund processing
+
+### Prerequisites
+You need Support specialist level access to Stripe, ask Simon for access.
+
+### How to check usage
+Before doing a refund, check their usage. You can make a copy of [this posthog dashboard](https://us.posthog.com/project/2/insights/8nLWTLHu) and use organization id to review account usage
```

Comment:
(Super minor) possibly call this an insight, as that's consistent with our terminology.